### ISSUE TITLE
How to conditionally add aria-desribedby for error validation message

### ISSUE THREAD

**Issue by sripalreddyadamala** [angular/angular#24968]
Hi all,
            Is there a way where we can conditionally add aria-describedby attribute conditionally when error is generated. Now I am referering to username error id but my audits fails since error message is dynamically generated after on blur. Any help is much appreciated.

<input type="text" class="form-control" id="username"  aria-invalid="false" aria-describedby="username_error_message formControlName="username" [(ngModel)]="model.username" required />
                <div *ngIf="(formSubmitted && loginForm?.controls?.username?.errors?.required) || (loginForm?.controls?.username?.touched && loginForm?.controls?.username?.errors?.required)"
                  class="help-block"  role="alert">
                  <span [innerHTML]="login_page_username_validation_msg" id="username_error_message"></span>
                </div>

**Comment by ericmartinezr** [comment]
Hello, we reviewed this issue and determined that it doesn't fall into the bug report or feature request category. This issue tracker is not suitable for support requests, please repost your issue on [StackOverflow](http://stackoverflow.com/) using tag `angular`.

If you are wondering why we don't resolve support issues via the issue tracker, please [check out this explanation](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-got-a-question-or-problem).

**Comment by kara** [comment]
Sorry, this doesn't look like a bug or a feature request, so I have to close this. Try using StackOverflow for help on attribute bindings, or our angular/angular gitter is very active / helpful.
